### PR TITLE
chore: allow writers to merge

### LINKED ISSUE #31: As a distribution merger, I can't merge https://github.com/mage-os/mageos-composer-root-update-plugin/pull/2

### PR DESCRIPTION
**Pull request by Vinai**
I'm not sure if `push_restrictions = []` has the desired effect, or if it shouldn't be declared at all.  

Fixes #31

### CODE REVIEW

**Comment by damienwebdev** [comment]
I think it's more appropriate to remove the setting @Vinai as @sprankhub mentioned in #31

**Comment by Vinai** [comment]
@damienwebdev I've removed the line as suggested.

**Comment by Vinai** [comment]
@sprankhub Can you enlighten me why removing the line makes the linter fail?

File: main.tf [hunk 1]
```diff
@@ -110,7 +110,6 @@ resource "github_branch_protection" "repositories" {
   for_each          = var.repositories
   repository_id     = github_repository.repositories[each.key].node_id
   pattern           = "*"
```
Comment (DavidLambauer) [inline main.tf:112]:
```suggestion
  pattern = "*"
```

@Vinai

Comment (Vinai) [reply main.tf:112]: Ah, thank you!

Comment (Vinai) [reply main.tf:112]: It still doesn't like me.

**Review by damienwebdev — APPROVED** [review]
lgtm

**Comment by damienwebdev** [comment]
@Vinai or @sprankhub please merge so that we can be freeeeeeeeeeee!

**Pull request merged**